orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -102,5 +103,16 @@ def test_min(self):
     x = cpu_tensor.min()
     assert torch.allclose(x, y.cpu())
 
+  def test_print_ort_tensor(self):
+    device = self.get_device()
+    cpu_tensor = torch.Tensor([[1, 1, 1], [1, 1, 1], [1, 1, 1]])
+    ort_ones = cpu_ones.to(device)
+    try:
+      print("Print ORT Tensor: ", ort_ones)
```

Comment:
we can try catch the print output use this trick to verify the print result:
https://github.com/microsoft/onnxruntime/blob/abjindal/create_print_ort_tensor_test/orttraining/orttraining/eager/test/ort_eps_test.py#L18